triton/examples/mpi/hello.rst

```diff
@@ -0,0 +1,57 @@
+The following example uses example codes stored in
+`the hpc-examples-repository <https://github.com/AaltoSciComp/hpc-examples>`_.
+You can get the repository with the following command::
+
+  git clone https://github.com/AaltoSciComp/hpc-examples/
+
+Loading module::
+
+  module load gcc/8.4.0      # GCC
+  module load openmpi/4.0.5  # OpenMPI
```

Comment:
Shouldn't the OpenMPI, if it is kind of linked to a compiler anyways, automatically load the corresponding compiler? Just wondering.